### PR TITLE
Separate playbooks for installation and configuration

### PR DESCRIPTION
**Pull request by motaparthipavankumar**
- Separated Keycloak installation and configuration.
- Updated README.md as per current changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,12 +45,35 @@ rhsso_rhn_id: '<sso_product_id>'
...
+
 * [`keycloak_realm`](https://github.com/ansible-middleware/keycloak/blob/main/roles/keycloak_realm/README.md): role for configuring a realm, with clients and users, in an installed service.
 
+### Example configuration command
+
+Execute the following command from the source root directory
+
+```
+ansible-playbook -i ansible_hosts -e @rhn-creds.yml playbooks/keycloak.yml -e keycloak_admin_password=<changeme> -e keycloak_realm=test`
+```
```
Comment (guidograzioli) [inline README.md:77]: Please add a note here telling ansible_hosts is the inventory containing and keycloak instance, and add a few lines showing the example inventory for running against localhost (the file below)

**Review by guidograzioli — COMMENTED** [review]
Please address my comment (better to show the example inventory only in README, instead of adding the file to the repo)

File: README.md (hunk 1 above)
Comment (motaparthipavankumar) [reply README.md:85]: Updated the README.md accordingly commit [link](https://github.com/ansible-middleware/keycloak/pull/5/commits/c78aaece81365f0dff91ed64735e6c5ab677858d)

**Comment by guidograzioli** [comment]
LGTM

**Pull request merged**